doc/options.yaml

```diff
@@ -432,6 +432,32 @@ useLinResMonitor:
     Enables the linear residual monitor for the NK and ANK solvers.
     The output should look like the linear residual output from the adjoint solver.
 
+useDissContinuation:
+  desc: >
+    Use dissipation-based continuation.
+    This only applies to the ``central plus scalar dissipation`` :py:data:`discretization`.
+    This adds second order dissipation near shocks at the initial stages of the solution, which helps converge supersonic flows.
```

Comment:
Should we be more explicit about the fact that this should not be used on "conventional" sub- and transonic conditions?